_locales/es/messages.json

```diff
@@ -0,0 +1,127 @@
+{
+	"name":
+	{
+		"message": "Pestañas guardadas",
+		"description": "Extension name. Displayed in the manifest and pane header"
+	},
+	"description":
+	{
+		"message": "Classic Microsoft Edge \"Tabs Aside\" feature",
+		"description": "Extension description"
+	},
+	"author":
+	{
+		"message": "Michael \"XFox\" Gordeev",
+		"description": "Author name"
+	},
+	"options":
+	{
+		"message": "Opciones",
+		"description": "Alternative text for options button in the pane"
+	},
+	"closePanel":
+	{
+		"message": "Cerrar",
+		"description": "Alternative text for close panel button"
+	},
+	"loadOnRestore":
+	{
+		"message": "Cargas pestañas al restaurar",
```

Comment:
Possible typo. GT suggests me to change it to "Carga**r** pestañas al restaurar"

Reply:
That's a typeo, my bad